tendermint/src/lite/error.rs

```diff
@@ -0,0 +1,73 @@
+//! All error types tied to the light client.
+
+use crate::Hash;
+use anomaly::{BoxError, Context};
+use std::time::{SystemTime, SystemTimeError};
+use thiserror::Error;
+
+/// The main error type verification methods will return.
+/// See [`ErrorKind`] for the different kind of errors.
+pub type Error = anomaly::Error<ErrorKind>;
+
+#[derive(Clone, Debug, Error)]
+pub enum ErrorKind {
```

Comment:
```suggestion
pub enum Kind {
```
Prefixes which are equivalent of the mod name are superfluous. This reads nicely: `error::Kind`.

Reply:
Good point! Addressed (needed to change this in other places too).